TextToSpeech/TextToSpeechImplementation.cpp

```diff
@@ -62,6 +65,17 @@ namespace Plugin {
         JsonObject config;
         config.FromString(service->ConfigLine());
 
+        InputValidation::Instance().addValidator("ttsendpoint", ExpectedValues<std::string>("^https?://[a-zA-Z0-9]+.*"));
+        InputValidation::Instance().addValidator("ttsendpointsecured", ExpectedValues<std::string>("^https://[a-zA-Z0-9]+.*"));
+        InputValidation::Instance().addValidator("double_str", ExpectedValues<std::string>("^-?[0-9]+(\\.[0-9]+)?"));
+        InputValidation::Instance().addValidator("speechid", ExpectedValues<uint32_t>(1, UINT32_MAX));
+        InputValidation::Instance().addValidator("text", ExpectedValues<std::string>("^[a-zA-Z0-9]+"));
+        InputValidation::Instance().addValidator("speechrate", ExpectedValues<std::string>({"slow", "medium", "fast", "faster", "fastest"}));
```

Comment:
May I know if any of these values are expected to be sent in uppercase from app?

Reply:
I am not sure if the app sends in uppercase.